_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,428 @@
+
+```
+npm install -g opensearch-reporting-cli-1.0.0.tgz
+```
+
+## Creating and requesting a visualization report
+
+First, you need to get the URL for the visualization that you want to download as an image file or PDF.
+
+To generate a visualization report, you need to specify the Dashboards URL.
+
+Open the visualization for which you want to generate a report, and select **Share >  Permalinks > Generate link as Shapshot > Short URL > Copy link**, as shown in the following image.
+
+![Copy link]({{site.url}}{{site.baseurl}}/images/dashboards/dash-url.png)
+
+You will need to add the URL with the `-u` argument when you request the report in the CLI.
+
+### Example: Requesting a PNG file
+
+The following command requests a report in PNG format with basic authentication and sends the report to an email address using Amazon SES:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:1234 -e ses -s <email address>  -r <email address> -f png
+```
+
+### Example: Requesting a PDF file
+
+The following command requests a PDF file and specifies the recipient's email address:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:Test@1234 -e ses -s <email address> -r <email address> -f pdf
+```
+
+Upon success, the file will be sent to the specified email address. The following image shows an example PDF report.
+
+![PDF example]({{site.url}}{{site.baseurl}}/images/dashboards/cli-pdf-report.png)
+
+### Example: Requesting a CSV file
+
+The following command generates a report that contains all table content in CSV format and sends the report to an email address using Amazon SES transport:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -f csv -a basic -c admin:Test1234 -e ses -s <email address> -r <email address>
+```
+
+Upon success, the email will be sent to the specified email address with the CSV file attached.
+
+## Automatically run reports with a cron job
+
+You can run reports automatically by setting up a cron job. OpenSearch Reporting CLI supports any cron job use case, such as setting a specified time of day.
+
+### Prerequisites
+
+You need a machine with cron installed and root access privileges.
+
+### Step 1: Install the reporting CLI
+
+Install the Reporting CLI by running the following command:
+
+```
+npm i @opensearch-project/opensearch-reporting-cli
+```
+
+### Step 2: Specify the details for the report
+
+Open the crontab editor by running the following command:
+
+```
+crontab -e
+```
+In the crontab editor, enter the report request. For example, the following example shows a cron report that runs every day at 8am.
+
+```
+0 8 * * * opensearch-reporting-cli -u https://playground.opensearch.org/app/dashboards#/view/084aed50-6f48-11ed-a3d5-1ddbf0afc873 -e ses -s <sender_email> -r <recipient_email>
```

Comment:
unrelated, is playground down? Playground link doesn't work on my end

Reply:
url works for me

Reply:
works for me too.
![Screen Shot 2023-02-03 at 4 28 40 PM](https://user-images.githubusercontent.com/88908598/216734878-02085560-56d3-4cb0-b445-8b8273689d38.png)